concent_api/core/models.py

```diff
@@ -288,6 +314,10 @@ class Meta:
 
     task_id                     = CharField(max_length = MESSAGE_TASK_ID_MAX_LENGTH)
 
+    computation_deadline        = DateTimeField(blank = True, null = True)
```

Comment:
should those new fields be nullable?

Reply:
Definitely not.

But Przemek mentioned that he's still working on `size` in TTC so maybe Golem is putting `None` there?

Reply:
Actually, this is `StoredMessage` and it may contain various types of messages, not only `TaskToCompute`. They do not have these fields. So the fields should be nullable after all.